.github/workflows/get-versions.yml

```
@@ -0,0 +1,46 @@
+#    This workflow exists to avoid using hardcoded versions of the toolchain in different places.
+#    In an ideal world, top-level constants (not yet implemented) or workflow-level environment variables
+#    would be used, but they are not supported in all needed places.
+
+name: get rust versions
+on:
+    workflow_call:
+        outputs:
+            stable-version:
+                value: ${{ jobs.get_versions.outputs.stable-version }}
+            nightly-version:
+                value: ${{ jobs.get_versions.outputs.nightly-version }}
+            old-version:
+                value: ${{ jobs.get_versions.outputs.old-version }}
+            matrix-version:
+                description: 'String like ["%stable%", "%nightly-yyyy-mm-dd%"].
+                         Added for `check` workflow, because there is no way to make a sequence from two outputs.'
+                value: ${{ jobs.get_versions.outputs.matrix-version }}
+
+env:
+   STABLE: "1.64.0"
+   NIGHTLY: "nightly-2022-10-01"
+   OLD: "1.56.0"
+
+jobs:
+    get_versions:
```

```suggestion
    get_rust_versions:
```
and drop `-version` suffix

---

> and drop `-version` suffix

Do you suggest changing the filename to `get-rust-versions.yml` and jobname `get_rust`? 
I got confused in similar ideas (https://github.com/intellij-rust/intellij-rust/pull/9625#discussion_r1011984885 ) and dropping `-versions` not`_versions`.

---

_clever thought comes later_
I think I guessed that it's about `..outputs.stable-version`